app/views/ops/_rbac_user_details.html.haml

```diff
@@ -150,37 +95,20 @@
           - else
             - select_groups = @edit[:new][:userid].blank? ? @edit[:new][:group] : groups.map(&:id)
             = select_tag('chosen_group',
-                         options_for_select(@edit[:groups], select_groups),
-                         :class    => "selectpicker",
-                         :multiple => true,
-                         :title    => "&lt;#{_('Choose one or more Groups')}&gt;",
-                         :style    => "overflow-x: scroll;")
+                        options_for_select(@edit[:groups], select_groups),
+                        :class    => "selectpicker",
+                        :multiple => true,
+                        :title    => "&lt;#{_('Choose one or more Groups')}&gt;",
+                        :style    => "overflow-x: scroll;")
             :javascript
               miqInitSelectPicker();
               miqSelectPickerEvent('chosen_group', "#{url}")
-    - if @edit
       .form-group
         %label.col-md-2.control-label
           = _("Selected Groups")
         .col-md-4
           = render :partial => "ops/rbac_group_selected"
-    - unless @edit
-      .form-group
-        %label.col-md-2.control-label
-          = _("Role")
-        .col-md-8
-          - if @record.current_group && @record.current_group.miq_user_role
-            - if role_allows?(:feature => "rbac_user_show")
-              - params = {:class => "pointer",
-                          :onclick => "miqOnClickSelectRbacTreeNode('ur-#{@record.current_group.miq_user_role.id}');",
-                          :title   => _("View this Role")}
-            - else
-              - params = {}
-            %i.ff.ff-user-role{params}
-            = link_to(@record.miq_user_role_name, "#", params)
-          - else
-            %p.form-control-static
-              = h(@record.miq_user_role_name)
-  %hr
-  - unless @edit
-    = render :partial => "rbac_tag_box"
+    %hr
+- else
+  = settings_users_summary(@record)
+  = render :partial => "rbac_tag_box"
```

Comment:
empty line after 114.

Reply:
@akhilkr128 made the changes . please check